cockatrice/src/client/settings/shortcuts_settings.h

```diff
@@ -562,9 +562,15 @@ class ShortcutsSettings : public QObject
                                      ShortcutGroup::Move_selected)},
         {"Player/aViewHand",
          ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Hand"), parseSequenceString(""), ShortcutGroup::View)},
-        {"Player/aSortHand", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Sort Hand"),
-                                         parseSequenceString("Ctrl+Shift+H"),
-                                         ShortcutGroup::View)},
+        {"Player/aSortHandByName", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Sort Hand by Name"),
+                                               parseSequenceString(""),
```

Comment:
I think I'd like some more default shortcuts for these at some point, but we have only so many chords on a keyboard